proposals/dao/tip_121a.ts

```diff
@@ -0,0 +1,246 @@
+      _backupOracle: ethers.constants.AddressZero,
+      _invertOraclePrice: false,
+      _decimalsNormalizer: 0
+    },
+    24 * 3600, // 1 day
+    '20000000000000000', // small weight 2%
+    '980000000000000000', // large weight 98%
+    addresses.lusd,
+    addresses.dai,
+    addresses.daiHoldingPCVDeposit, // send DAI to Compound DAI deposit, where it can then be dripped to PSM
+    ethers.utils.parseEther('100000') // minimum size of a pool which the swapper is used against
+  );
+
+  await lusdToDaiSwapper.deployed();
+  logging && console.log('LUSD to DAI swapper deployed to:', lusdToDaiSwapper.address);
+
+  // Create a liquidity bootstrapping pool between LUSD and DAI
+  const lbpFactory = await ethers.getContractAt(
+    'ILiquidityBootstrappingPoolFactory',
+    addresses.balancerLBPoolFactoryNoFee
+  );
+
+  const tx: TransactionResponse = await lbpFactory.create(
+    'LUSD->DAI Auction Pool', // pool name
+    'apLUSD-DAI', // lbp token symbol
+    [addresses.lusd, addresses.dai], // pool contains [LUSD, DAI]
+    [ethers.constants.WeiPerEther.mul(98).div(100), ethers.constants.WeiPerEther.mul(2).div(100)], // initial weights 5%/95%
+    ethers.constants.WeiPerEther.mul(5).div(10_000), // 0.05% swap fees
+    lusdToDaiSwapper.address, // pool owner = fei protocol swapper
+    true
+  );
+
+  const txReceipt = await tx.wait();
+  const { logs: rawLogs } = txReceipt;
+  noFeeLusdDaiLBPAddress = `0x${rawLogs[rawLogs.length - 1].topics[1].slice(-40)}`;
+  poolId = rawLogs[1].topics[1];
+
+  logging && console.log('LBP Pool deployed to: ', noFeeLusdDaiLBPAddress);
+  logging && console.log('LBP Pool Id: ', poolId);
+
+  // Initialise the LBP swapper with the pool address
+  const tx2 = await lusdToDaiSwapper.init(noFeeLusdDaiLBPAddress);
+  await tx2.wait();
+
+  // Deploy a lens to report the swapper value
+  const BPTLensFactory = await ethers.getContractFactory('BPTLens');
+  const lusdToDaiLensDai = await BPTLensFactory.deploy(
+    addresses.dai, // token reported in
+    noFeeLusdDaiLBPAddress, // pool address
+    addresses.chainlinkDaiUsdOracleWrapper, // reportedOracle - DAI
+    addresses.chainlinkLUSDOracleWrapper, // otherOracle - LUSD
+    false, // feiIsReportedIn
+    false // feiIsOther
+  );
+  await lusdToDaiLensDai.deployTransaction.wait();
+
+  logging && console.log('BPTLens for DAI in swapper pool: ', lusdToDaiLensDai.address);
+
+  const lusdToDaiLensLusd = await BPTLensFactory.deploy(
+    addresses.lusd, // token reported in
+    noFeeLusdDaiLBPAddress, // pool address
+    addresses.chainlinkLUSDOracleWrapper, // reportedOracle - LUSD
+    addresses.chainlinkDaiUsdOracleWrapper, // otherOracle - DAI
+    false, // feiIsReportedIn
+    false // feiIsOther
+  );
+  await lusdToDaiLensLusd.deployTransaction.wait();
+
+  logging && console.log('BPTLens for LUSD in swapper pool: ', lusdToDaiLensLusd.address);
+
+  // Deploy gOHM otc escrow
+  const ohmEscrowUnwind = await (
```

Comment:
Do we want to ask Olympus to give this a once over and check they're good with it? Presume we'll send them the contract deploy params/verified Etherscan link anyhow

Reply:
I'll ping them in discord, but they had a look at the PR